blog/_posts/2016-09-28-almost-done-jammin.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Almost done jammin'
+layout: post
+created_at: Mon Sep 25 2016
+permalink: blog/2016-09-28-goodbye-jam
```

Comment:
could you make the permalink consistent with the file name? (meaning: using the same name for both)